app/controllers/objects_controller.rb

```diff
@@ -20,10 +20,10 @@ def checksum
 
   # return the checksums and filesize for a list of druid (supplied with druid: prefix)
   # note: this is deliberately allowed to be a POST to allow for a large number of druids to be passed in
-  # GET OR POST /objects/checksums?druids[]=druida&druids[]=druidb&druids[]=druidc
+  # GET OR POST /objects/checksums?druids[]=druid1&druids[]=druid2&druids[]=druid3
   def checksums
     unless druids.present?
-      render(plain: '400 bad request', status: :bad_request)
+      render(plain: "400 bad request - at least one of these druids is not found: #{druids}", status: :bad_request)
```

Comment:
this message strikes me as slightly misleading... i think `druids.present?` is checking to see that the caller provided a druid list, not that all of the druids are present in pres cat (hence the `400` instead of the `404`, iirc).

Reply:
https://guides.rubyonrails.org/active_support_core_extensions.html#blank-questionmark-and-present-questionmark

> The following values are considered to be blank in a Rails application:
    * nil and false,
    * strings composed only of whitespace (see note below),
    * empty arrays and hashes, and
    * any other object that responds to empty? and is empty.
...
The method present? is equivalent to !blank?.

Reply:
I addressed this one - thanks for catching that.